db/import.go

```diff
@@ -149,13 +156,19 @@ func (db *Database) importDoc(docid string, body Body, isDelete bool, existingDo
 					Expiry: expiry,
 				}
 
+				if doc.inlineSyncData {
+					existingDoc.Body, _ = doc.MarshalBodyAndSync()
```

Comment:
Why is it safe to ignore error here and below?

Reply:
Its not. Good spot.